lib/src/commands/test/test.dart

```diff
@@ -113,6 +116,9 @@ This command should be run from the root of your Flutter project.''',
     final excludeFromCoverage = _argResults['exclude-coverage'] as String?;
     final randomOrderingSeed =
         _argResults['test-randomize-ordering-seed'] as String?;
+    final randomSeed = randomOrderingSeed == 'random'
+        ? Random().nextInt(4294967295).toString()
```

Comment:
Just curious, can't we just call nextInt with passing a limit?

Reply:
What do you mean? Can you provide a suggestion?

Reply:
My bad, I confused this with `nextDouble` which is called without a limit, here we need to pass a limit!